matlab/doc/matlab_interface_for_apache_arrow_design.md

```diff
@@ -131,33 +119,43 @@ Alternatively, they could directly convert from an existing MATLAB `table` to an
 
 >> Density = [10.2; 20.5; 11.2; 13.7; 17.8]; 
 
->> T = table(Weight, Radius, Density); % Create a MATLAB table 
+% Create a MATLAB `table`
+>> T = table(Weight, Radius, Density);
 
->> AT = arrow.matlab2arrow(T); % Create an arrow.Table 
+% Create an `arrow.tabular.Table` from the MATLAB `table`
+>> AT = arrow.table(T);
 ```
-To serialize the `arrow.Table`, `AT`, to a file (e.g. Feather) on disk, the user could then instantiate an `arrow.FeatherTableWriter`. 
+
+To serialize the `arrow.Table`, `AT`, to a file (e.g. Feather) on disk, the user could then instantiate an `arrow.internal.io.feather.Writer`. 
 
 ###### Example Code: 
 ``` matlab
->> featherTableWriter = arrow.FeatherTableWriter(); 
-
->> featherTableWriter.write(AT, "data.feather"); 
+% Make an `arrow.tabular.RecordBatch` from the `arrow.tabular.Table` created in the previous step
+>> recordBatch = arrow.recordBatch(AT);
+>> filename = "data.arrow";
```

Comment:
To make it clear that this is Feather V1 and *NOT* Feather V2 (i.e. the Arrow IPC format), I think it would be better to use the extension `.feather` here.

```suggestion
>> filename = "data.feather";
```